src/plugins/event-plugins/__tests__/WebVitalsPlugin.test.ts

```diff
@@ -52,7 +64,7 @@ describe('WebVitalsPlugin tests', () => {
         record.mockClear();
     });
 
-    test('When web vitals are present then events are recorded', async () => {
+    test('When web vitals are present then LCP is recorded with attributions', async () => {
```

Comment:
question: Do we need unit tests to cover case where attributions could not be collected?

Reply:
Yes, we need "when [web vital] does not have attribution then event is recorded without attribution". Thanks

Reply:
Correction: this test is not needed. Attributions object will always exist. Some properties are optional, such as `LCP.element` and all of the CLS fields. These fields will be defined as `{element: undefined...}` and dropped entirely during `JSON.stringify()`.